src/user.js

```diff
@@ -20,9 +20,9 @@ class User {
     if(recipe !== undefined) this.recipesToCook.push(recipe);
   };
 
-  filterRecipes(type) {
+  filterRecipes(type, recipeList) {
```

Comment:
I think making the filter dynamic is going to save us time in the future. Well done!